src/lib.rs

```diff
@@ -103,17 +103,16 @@ impl Build {
             .arg("no-comp")
             .arg("no-zlib")
             .arg("no-zlib-dynamic")
-            // This actually fails to compile on musl (it needs linux/version.h
-            // right now) but we don't actually need this most of the time. This
-            // is intended for super-configurable backends and whatnot
-            // apparently but the whole point of this script is to produce a
-            // "portable" implementation of OpenSSL, so shouldn't be any harm in
-            // turning this off.
-            .arg("no-engine")
             // MUSL doesn't implement some of the libc functions that the async
             // stuff depends on, and we don't bind to any of that in any case.
             .arg("no-async");
 
+        if target.contains("musl") || target.contains("android") || target.contains("windows") {
+            // This actually fails to compile on musl (it needs linux/version.h
+            // right now) but we don't actually need this most of the time.
```

Comment:
Could this comment be updated as to why this option is skipped for each of the platforms here?

Reply:
I tried to make windows pass CI but failed. Comments is added